e2e/test_dataset_manager.py

```diff
@@ -41,6 +48,25 @@ def tearDown(self):
         self.dataset_manager.remove_dataset()
 
 
+class PartitionedDatasetManagerPropertiesTestCase(DatasetManagerTestCase):
```

Comment:
Those tests are not necessary and are not e2e

Reply:
e2e tests are slow and hard to run(you need a project). Unit tests are fine. Maybe @ewebed can add one e2e test, but I'm not sure if that a good direction.

Reply:
Ok, but those tests should not be in /e2e directory. But imo we need e2e test